AssetManagementTestProject/Services/AuthorizationService.cs

```diff
@@ -11,25 +11,28 @@ public class AuthorizationService
     /// <summary>
     /// API Authorize
```

Comment:
Get all the code in the `CreateUserService.cs` into 1 class and name it `AssetManagementAPIServices.cs`